spring-boot-starters/spring-boot-starter-test/pom.xml

```diff
@@ -40,6 +40,10 @@
 			<artifactId>junit</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>org.xmlunit</groupId>
```

Comment:
Could you move this down a bit please? We try to keep the entries in alphabetical order using groupId first and then artifactId if required